src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionSet.cs

```diff
@@ -22,7 +22,7 @@ internal class UnifiedSuggestedActionSet
 
         public object? Title { get; }
 
-        public CodeActionPriority Priority { get; }
+        public CodeActionPriorityInternal Priority { get; }
```

Comment:
❓  Changing the property types seems unnecessary. Can we follow the pattern of `SymbolDisplayPartKind`, where the internal values are just provided by an internal type, and the value coersion is performed at the use site?